boards/acd52832/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_acd52832
+ * @{
+ *
+ * @file
+ * @brief       Board initialization for the nRF52 DK
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "board.h"
+
+void board_init(void)
+{
+  /* initialize the boards LEDs */
+  NRF_P0->DIRSET = (LED0_MASK);
+  NRF_P0->OUTSET = (LED0_MASK);
+  NRF_P0->OUTSET = (LED0_MASK);
```

Comment:
indention is off by two spaces

Reply:
also: the line above is duplicated...

Reply:
@haukepetersen Taken over from boards/nrf52dk/board.c

Reply:
then you can fix that file as well :-)